src/main/java/com/openlattice/edm/PostgresEdmManager.java

```diff
@@ -186,20 +188,54 @@ public void revoke(
         }
     }
 
-    public Set<UUID> getAllPropertyTypeIds() {
-        String getAllPropertyTypeIds = String.format("SELECT id from %1$s", PROPERTY_TYPES);
+    /**
+     * Duplicate of
+     * {@link com.openlattice.data.storage.PostgresEntityDataQueryService#getLinkingEntitySetIdsOfEntitySet(UUID)}
+     */
+    public Iterable<EntitySet> getAllLinkingEntitySetsForEntitySet( UUID entitySetId ) {
```

Comment:
Use Hazelcast for this as it will be significantly faster and keep load of the database. If you add an index in the entity set mapstore for the linked entity sets field there is even support for indexing array types.

Reply:
Will do.